File: backend/src/main/java/ar/edu/um/backend/service/dto/AsientoBloqueoResponseDTO.java
package ar.edu.um.backend.service.dto;
import lombok.*;
import java.io.Serializable;
import java.util.List;
/**
 * Respuesta del bloqueo de asientos (Payload 6).
 *
 * Indica si el bloqueo fue exitoso y el estado final
 * de cada asiento solicitado.
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class AsientoBloqueoResponseDTO implements Serializable {

    private Long eventoId;
    private boolean resultado;
    private String descripcion;
    private List<AsientoEstadoDTO> asientos;
}

File: frontend/composeApp/src/commonMain/kotlin/ar/edu/um/tpfinal/dto/AsientoBloqueoResponseDTO.kt
package ar.edu.um.tpfinal.dto

import kotlinx.serialization.Serializable

@Serializable
data class AsientoBloqueoResponseDTO(
    val eventoId: Long,
    val resultado: Boolean,
    val descripcion: String,
    val asientos: List<AsientoEstadoDTO>
)
